account_statement_import_camt/models/account_statement_import.py

```diff
@@ -32,7 +32,7 @@ def _parse_file(self, data_file):
                 return currency, account_number, transactions
             # pylint: disable=except-pass
             except (zipfile.BadZipFile, ValueError):
-                pass
+                _logger.exception("BadZipfile exception")
```

Comment:
Why is an exception logged now instead of ignored? This code will run on any file that is not a valid camt file, i.e. any file of any other format supported by the import framework.